infrastructure/main.tf

```diff
@@ -1,6 +1,6 @@
 
 locals {
-  db_connection_options  = "?ssl=true"
```

Comment:
Looks like we need both of flags to be present

Reply:
no we shouldn't, the sslmode is require though I'm pretty sure not required :tongue:

Reply:
ah. just need to heal jenkins now